spec/jobs/issue_badge_job_spec.rb

```diff
@@ -15,22 +19,30 @@
       badge
     end
 
-    it "should skip as user is not enrolled" do
-      described_class.perform_now(achievement:)
-      expect(Credly::Badge).not_to have_received(:issue)
-    end
-
     context "when the user is enrolled" do
       before do
         user_programme_enrolment
       end
 
       it "calls Credly::Badge.issue" do
-        described_class.perform_now(achievement:)
+        described_class.perform_now(achievement: f2f_achievement)
         expect(Credly::Badge).to have_received(:issue)
       end
     end
 
+    context "should not assign badge when" do
+      it "user is not enrolled" do
+        described_class.perform_now(achievement: f2f_achievement)
+        expect(Credly::Badge).not_to have_received(:issue)
+      end
+
+      it "when achievement is not F2F" do
+        user_programme_enrolment
+        described_class.perform_now(achievement: community_achievement)
+        expect(Credly::Badge).not_to have_received(:issue)
+      end
```

Comment:
Just to clarify for my own understanding, are badges never awarded for community achievements - is there always a face to face component?

Reply:
Yeah the badges only should be issued for completing a CPD course (so face-to-face, remote, or online), but not for community activities.